src/backend/core/api/serializers.py

```diff
@@ -228,5 +228,5 @@ class InvitationSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = models.Invitation
-        fields = ["email", "team", "role", "issuer"]
-        read_only_fields = ["team", "issuer"]
+        fields = ["created_at", "email", "team", "role", "issuer", "is_expired"]
+        read_only_fields = ["created_at", "team", "issuer", "is_expired"]
```

Comment:
```suggestion
        fields = ["id", "email", "team", "role", "issuer", "is_expired", "created_at"]
        read_only_fields = ["id", "team", "issuer", "is_expired", "created_at"]
```

Reply:
we need the ID to perform delete operation.

Reply:
Yes, needed

Reply:
Done !